llvm/test/CodeGen/RISCV/rvv/vl-opt-op-info.mir

```diff
@@ -513,6 +513,36 @@ body: |
     %y:vr = PseudoVNSRL_WV_MF2 $noreg, $noreg, %x, 1, 3 /* e8 */, 0
 ...
 ---
+name: vfnop_vs2
```

Comment:
vfnop?

Reply:
Above is vnop for vector narrowing op. This is vfnop for vector floating narrowing op.

Reply:
Well that' just confusing because "nop" is also the name of the "no operation" instruction.

Reply:
How about I rename vfnop and vnop in a follow up pr to vfnarrow_op and vnarrow_op ?

Reply:
sure